### DISCUSSION TITLE
Run server as windows service

### DISCUSSION THREAD

**Discussion by mmiacca** [cutelyst/cutelyst#354] [Q&A]
Dear, anyone was try to write a windows service using cutelyst server? i have no problems with exe running from terminal, but i have no success triyng to run it into a windows service (using qtservice).
Congratulations Daniel, incredible work!!!!

**Comment by cristeab** [comment]
How you run the server from the terminal ? I am getting this error:
"3040:3000 cutelyst.server[critical] Could not load application: "Cannot load library C:\\NdiAudioRouter\\app.dll: The specified module could not be found."", but the DLL exists. I have managed to run the server on Linux and I have copied the same libraries in Windows.

**Comment by Zmrd13** [comment]
QtService works well with the server (I just moved the internals of main.cpp with server initialization to the service startup method). There may be an issue with your app's incorrect currentPath. By default, services run in the system32 directory, not in the.exe path. And don't use server.exec, it calls application::exec() inside, which the service does too
